### ISSUE TITLE
Dashboard: hover actions to convert paragraph type

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#467]
I love the click-to-complete on the Dashboard and the hover states for rescheduling. 
Often, I want to cancel a task or convert it to a bullet so the text is there but it's not a task anymore.
Can you add a fancy hover to the noteplanstate icon? so clicking does what it does today, but hovering gives you the option to change the state to another type?
```
['open', 'done', 'list', 'checklist', 'checklistDone','text']
```
Obviously the cleanest version would omit the current state from the hover list

**Comment by jgclark** [comment]
I see the purpose entirely, but honestly I'm more keen not to do this, than to do it.
There are many moving parts required in both the window UI and DataStore, and the comms channel between them, and it already feels fragile and difficult for me to maintain, let alone anyone else. 
I'm also not happy yet at the tooltip rendering; it follows constraints which are invisible to users, which makes for inconsistent layout. I'd want at least to get that resolved first.

**Comment by jgclark** [comment]
@dwertheimer is this still a request from you? It would be much easier to add now.

**Comment by dwertheimer** [comment]
Yes, this would be great. I can't believe that it's been three years that we've been talking about dashboard.
